network/p2p/throttler_test.go

```diff
@@ -0,0 +1,114 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package p2p
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"golang.org/x/time/rate"
+
+	"github.com/ava-labs/avalanchego/ids"
+)
+
+func TestThrottler(t *testing.T) {
+	type call struct {
+		tokens      int
+		expectedErr bool
```

Comment:
```suggestion
		expectedErr error
```
And then use `require.ErrorIs` in the below test.